pkgs/racket-test/tests/stxparse/test.rkt

```diff
@@ -50,22 +50,92 @@
 (terx (1 (2 3)) (_:one _:two) "expected one")
 (terx ((1) 2) (_:one _:two) "expected two")
 
-;; datum patterns
-(tok 1 1
-     'ok)
-(tok 1 _
-     #t
-     #:pre [2] #:post [])
-(tok "here" "here"
-     'ok
-     #:pre ["there"] #:post [])
-(tok #f #f
-     'ok
-     #:pre [#t 0] #:post [_])
-
-(terx 1 2 "literal 2")
-(terx (1 2) 1 "literal 1")
-(terx (1 2) (1 1) "literal 1")
+(test-case "datum patterns"
```

Comment:
don't nest test cases within other test cases (rackunit needs a non-delaying version of test-suite)